### PR TITLE
DIFM Lite: Redesigned In-progress screen

### PR DESCRIPTION
**Pull request by aneeshd16**
#### Changes proposed in this Pull Request

* Redesigned DIFM-L in-progress screen according to the new design: 2b998-pb/#plain

#### Testing instructions

* Log in and switch to a site that has the `difm-lite-in-progress` sticker. (It can be toggled from the Blog RC)
* Verify that the design matches the one in the link above.

image

### CODE REVIEW

File: client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx [hunk 1]
```diff
@@ -47,36 +47,29 @@ function DIFMLiteInProgress( { siteId }: DIFMLiteInProgressProps ): React.ReactE
...
+				domain: domainName,
+			} )
+		);
 	};
 
 	return (
 		<div>
 			<EmptyContent
-				title={ translate( 'Hang on! Our experts are building your site.' ) }
+				title={ translate( 'Our experts are building your site' ) }
```
Comment (a8ci18n) [inline client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx:60]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Review by dzver — APPROVED** [review]
LGTM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7138771

Thank you @aneeshd16 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7138771).

**Pull request merged**